karma.conf.js

```diff
@@ -36,7 +36,7 @@ module.exports = function (config) {
       check: {
         global: {
           statements: 100,
-          branches: 91,
+          branches: 100,
```

Comment:
With a misfire on a rebase I reverted the code coverage changes, I will rebase this PR when #44 gets re-merged in